components/Topbar/Topbar.tsx

```diff
@@ -22,22 +22,24 @@ export const Topbar = () => {
     }
   };
   const SUBMIT_PAGE = '/projects/submit';
+  const NON_DESKTOP_ROUTES = ['/', '/404'];
   return (
     <div className="z-50 border-b border-gray-800 bg-gray-900">
       <nav className="m-auto flex h-[5.46rem] max-w-screen-xl justify-between p-4 py-5">
         <div className="flex items-center gap-10">
           <Logo />
-          {router.pathname !== '/' && <DesktopRoutes />}
+          {!NON_DESKTOP_ROUTES.includes(router.pathname) && <DesktopRoutes />}
```

Comment:
could you please rename it MOBILE_ROUTES? non_desktop_routes kinda makes it too long imo